doc/api/events.md

```diff
@@ -1485,11 +1485,10 @@ target.removeEventListener('foo', handler, { capture: true });
 added: v14.5.0
 -->
 
-* `event` {Object|Event}
+* `event` {Event}
 
 Dispatches the `event` to the list of handlers for `event.type`. The `event`
-may be an `Event` object or any object with a `type` property whose value is
-a `string`.
+must be an `Event` object.
```

Comment:
I'm not sure we need this sentence anymore.
```suggestion
```

Reply:
By removing `Object`, I think it is implied that `event` must be an `Event`. So, it's safe to remove.